contrib/babelfishpg_tsql/src/pltsql.h

```diff
@@ -34,6 +34,7 @@
 
 #include "dynavec.h"
 #include "dynastack.h"
+#include "../../contrib/babelfishpg_common/src/babelfishpg_common.h"
```

Comment:
why is this required? We already have callbacks from TSQL to common (for collation etc.), how do they work?

Reply:
This is needed so that we don't have to declare the rendezvous variable structure at two places. Other examples [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/contrib/babelfishpg_tds/src/include/tds_int.h#L37).

Reply:
fine for now but we need to think more. We have taken care of link time dependency but added a compile time dependency from TSQL extension to common extension. I guess such exchange should pass thorough engine as that is common to all extensions.